bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/config/config.xml

```diff
@@ -5,123 +5,186 @@
 	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0
 		https://openhab.org/schemas/config-description-1.0.0.xsd">
```

Comment:
```
[WARNING] Code Analysis Tool has found:
 0 error(s)!
 10 warning(s)
 10 info(s)
[WARNING] OH-INF.config.config.xml:[0]
Unused configuration reference with uri - netatmo:hum:config
[WARNING] OH-INF.config.config.xml:[0]
Unused configuration reference with uri - netatmo:configurable
[WARNING] OH-INF.config.config.xml:[0]
Unused configuration reference with uri - netatmo:virtual
[WARNING] OH-INF.config.config.xml:[0]
Unused configuration reference with uri - netatmo:temp:config
[WARNING] OH-INF.config.config.xml:[0]
Unused configuration reference with uri - netatmo:sum_rain:config
[WARNING] OH-INF.config.config.xml:[0]
Unused configuration reference with uri - netatmo:co2:config
[WARNING] OH-INF.config.config.xml:[0]
Unused configuration reference with uri - netatmo:device
[WARNING] OH-INF.config.config.xml:[0]
Unused configuration reference with uri - netatmo:timestamp:config
[WARNING] OH-INF.config.config.xml:[0]
Unused configuration reference with uri - netatmo:noise:config
[WARNING] OH-INF.config.config.xml:[0]
Unused configuration reference with uri - netatmo:pressure:config
```

Reply:
This is normal. These configuration elements are not referenced in any thing.xml file as this binding has his own ThingTypeProvider. I don't know how to avoid these warnings.

Reply:
ok